src/otx/core/engine/utils/instantiators.py

```diff
@@ -0,0 +1,56 @@
+from typing import List
```

Comment:
Ths file originates from the lightning-hydra-template. Please include the source and license information in the file header for proper attribution.

Reply:
Thanks, updated every file's license here https://github.com/openvinotoolkit/training_extensions/pull/2670/commits/53d8c6aab281488cd85d2a049e51109c6d1ab4a9